pages/price-feeds/use-real-time-data/sui.mdx

```diff
@@ -1,82 +1,124 @@
+# Pyth is locked into this specific `rev` because our package depends on Wormhole and is pinned to this version.
+[dependencies.Sui]
+git = "https://github.com/MystenLabs/sui.git"
+subdir = "crates/sui-framework/packages/sui-framework"
+rev = "041c5f2bae2fe52079e44b70514333532d69f4e6"
+````
 
+</Tabs.Tab>
+
+  <Tabs.Tab>
 ```sh copy
-yarn add @pythnetwork/pyth-sui-js
-```
+[dependencies.Pyth]
+git = "https://github.com/pyth-network/pyth-crosschain.git"
+subdir = "target_chains/sui/contracts"
+rev = "sui-contract-movement-m2-devnet"
 
-## Quickstart
+[dependencies.Wormhole]
+git = "https://github.com/pyth-network/pyth-crosschain.git"
+subdir = "target_chains/sui/vendor/wormhole_movement_devnet/wormhole"
+rev = "sui-contract-movement-m2-devnet"
 
-Pyth stores prices off-chain to minimize gas fees, which allows us to offer a wider selection of products and faster update times.
-See [Pull Updates](../pythnet-price-feeds/pull-updates) for more information about this approach.
-Typically, to use Pyth prices on chain,
-they must be fetched from the [Hermes API](https://hermes.pyth.network/docs). The `SuiPriceServiceConnection` class can be used to interact with these services,
-providing a way to fetch these prices directly in your code. The following example wraps an existing RPC provider and shows how to obtain
-Pyth prices and submit them to the network:
+# Pyth is locked into this specific `rev` because our package depends on Wormhole and is pinned to this version.
 
-```typescript copy
-import { SuiPriceServiceConnection } from "@pythnetwork/pyth-sui-js";
+[dependencies.Sui]
+git = "https://github.com/MystenLabs/sui.git"
+subdir = "crates/sui-framework/packages/sui-framework"
+rev = "041c5f2bae2fe52079e44b70514333532d69f4e6"
 
-const connection = new SuiPriceServiceConnection(
-  "https://hermes-beta.pyth.network"
-); // See Hermes endpoints section below for other endpoints
+````
 
-const priceIds = [
-  // You can find the ids of prices at https://pyth.network/developers/price-feed-ids
-  "0xf9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b", // BTC/USD price id in testnet
-  "0xca80ba6dc32e08d06f1aa886011eed1d77c77be9eb761cc10d72b7d0a2fd57a6", // ETH/USD price id in testnet
-];
+</Tabs.Tab>
+</Tabs>
 
-// In order to use Pyth prices in your protocol you need to submit the price update data to Pyth contract in your target
-// chain. `getPriceUpdateData` creates the update data which can be submitted to your contract.
+Pyth also provides a javascript SDK to construct transaction blocks that update price feeds:
 
-const priceUpdateData = await connection.getPriceFeedsUpdateData(priceIds);
-```
+<Tabs items={["npm", "yarn"]}>
+  <Tabs.Tab>
+    ```sh copy npm install --save @pythnetwork/pyth-sui-js ```
+  </Tabs.Tab>
+  <Tabs.Tab>```sh copy yarn add @pythnetwork/pyth-sui-js ```</Tabs.Tab>
+</Tabs>
 
-## On-chain prices
+## Write Contract Code
 
-### ⚠️ **_Important Note for Integrators_**
+<Callout type="warning" emoji="⚠️">
```

Comment:
I think this callout should come after the code snippet.

Reply:
This is pretty important piece of information, so I believe the callout should be before the code snippet.